openfaas/deno-http-template/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM austinrivas/deno-http-base:stable
+
+WORKDIR ${FUNC_HOME_DIR}
```

Comment:
Is this value inherited from the base image?

Reply:
Yes, the user is too.